packages/fastui/src/components/Iframe.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react'
+
+import { ClassName } from '../hooks/className'
+
+export interface IframeProps {
+  type: 'Iframe'
+  src: string
+  width?: string
+  height?: string
```

Comment:
I think these should allow `number` / `int`.

Reply:
i think they should also allow strings as sometimes the width and height (in css) are defined as px or em

Reply:
wait, just checked that the new code has an or hahaha. my bad